Sources/NIO/Embedded.swift

```diff
@@ -251,6 +251,37 @@ class EmbeddedChannelCore: ChannelCore {
     }
 }
 
+/// `EmbeddedChannel` is a `Channel` implementation that does neither do any
+/// actual IO nor has a proper eventing mechanism. The prime use-case for
```

Comment:
Nit: s/does neither do any/does neither any/

Reply:
thanks, done, sorry for the delay